### DISCUSSION TITLE
Using /Extras/LevelTemplate seems a bit risky.

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#3862] [Ideas]
The 'Extras' folder comes with the mindset that its contents are only extras and that the base game doesn't rely on them. The generation and deletion of item icons and outfit previews further reinforces this.

**Comment by SDGNelson** [comment]
The reason I moved a few things out of the Bundles folder was so that the asset reading thread can reasonably expect all of the files in the Bundles directory to be asset-related. Do you have a particular concern about the Extras directory, or just that it would mess up level creation if someone deleted it?

**Reply by Jdance-Media** [reply]
My only concern is that one may think it would be fine to delete LevelTemplate's contents, resulting in messing up level creation. Other than that, nothing else, as that particular folder originally started out as unnecessary extras.
